src/main/resources/db/changelog/v.0.2/db.changelog-user-role.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: 04042021-add-table-user_role
+      author: maksim_maevsky
+      changes:
+        - createTable:
+            tableName: user_role
```

Comment:
**user_role** is confusing.

```suggestion
            tableName: role
```

and change column prefixes as well.

Reply:
done